persistence/sql/migrations/templates/20210913095309_identity_recovery_tokens.up.fizz

```diff
@@ -0,0 +1,9 @@
+add_column("identity_recovery_tokens", "identity_id", "uuid", {"size": 36})
+{{ if not .IsSQLite }}
+  add_foreign_key("identity_recovery_tokens", "identity_id", {"identities": ["id"]}, {
+      "name": "identity_recovery_tokens_identity_id_fk_idx",
+      "on_delete": "CASCADE",
+      "on_update": "RESTRICT",
+  })
+{{ end }}
```

Comment:
You can add foreign keys in sqlite like so:

https://github.com/ory/kratos/blob/4bec9ef50f14f22342a311f09ba1b59cde47befc/persistence/sql/migrations/templates/20210410175418_network.up.fizz#L4

Reply:
Sorry missed that comment. I'll try to fix it today

Reply:
@aeneasr if I add a column this way the migrations don't generate properly. The identity_id column is added i one of the first migrations and then is being removed after some temp table is generated and data is copied there. This causes tests to fail since the identity_id column doesn't exist
`"sqlite create: table identity_recovery_tokens has no column named identity_id"`

Reply:
Could you share the full up/down migration?

Reply:
@aeneasr https://github.com/ory/kratos/pull/1750/commits/45ec59bd08509c775520bc2db59a12626c1c64b5

Reply:
did you run `migrations-render-replace`? Another good practice is to clear out all the migrations in the sql repository related to your change. if you change migration templates the output might be a file that should have been changed or deleted for example

Reply:
Did it help? :)

Reply:
sorry @aeneasr  no it didn't. I still get "sqlite create: table identity_recovery_tokens has no column named identity_id" . The only change that wasn't in the pr is in this file: 20210410175418000063_network.sqlite3.down.sql, but it doesn't matter.

Reply:
Ok, I'll take a look

Reply:
I just checked out your branch, ran `rm -rf persistence/sql/migrations/sql/* && make migrations-render-replace` and it worked. Not sure what's going on on your end?

Assuming tests pass, is this PR complete in your opinion?

Reply:
> I just checked out your branch, ran `rm -rf persistence/sql/migrations/sql/* && make migrations-render-replace` and it worked. Not sure what's going on on your end?
> 
> Assuming tests pass, is this PR complete in your opinion?

`make migrations-render-replace` works, but tests fail after that the same way. After regenerating migrations  the only difference between my pr and local changes is the file I mentioned in the previous comment.

As for the PR I would say it's finished.